deploy/swrs/transform/function/load_emission.sql

```diff
@@ -173,6 +173,7 @@ $function$
                 and _activity.sub_process_name = 'Flaring'
                 and _emission.gas_type = 'N2O')
             or (_fuel_mapping.fuel_type = 'Vented Natural Gas'
+                and _emission.gas_type = 'CH4'
```

Comment:
Not quite sure what's going on here. There's also some weird conditions above:
```
_fuel_mapping.fuel_type = 'Flared Natural Gas N2O'
and _fuel_mapping.fuel_type != 'Flared Natural Gas CH4'
and _fuel_mapping.fuel_type != 'Flared Natural Gas CO2'
```

Reply:
yeah, looks redundant and can be removed